HACKING.md

```diff
@@ -14,33 +14,26 @@ You'll also want to do some terminology mapping between what we say to users and
   * Attempt → Try: Try is a keyword in Elixir, and in a lot of other languages besides,
                    so the code calls a try an attempt.
   * Patch → PR: A Bors "Patch" corresponds to a GitHub "Pull Request."
-  * Aelita2 → Bors-NG: I renamed it without changing the name in the code.
```

Comment:
I didn't remove this line in my changes to this file since "aelita" is still used in some instances of the repo.